packages/manager/modules/pci/src/projects/project/instances/add/translations/Messages_fr_FR.json

```
@@ -30,11 +29,12 @@
   "pci_projects_project_instances_add_script_add_label": "Ajouter",
   "pci_projects_project_instances_add_privateNetwork_label": "Réseau privé attaché",
   "pci_projects_project_instances_add_privateNetwork_none": "Aucun",
+  "pci_projects_project_instances_add_privateNetwork_description": "Par défaut les instances Public Cloud possèdent un réseau public, vous pouvez ajouter un réseau privé.",
+  "pci_projects_project_instances_add_privateNetwork_add": "Créer un Private network",
```

I see it translated in other texts
```suggestion
  "pci_projects_project_instances_add_privateNetwork_add": "Créer un réseau privé",
```

---

Fixed.